src/AutoROM.py

```diff
@@ -197,6 +197,23 @@ def torrent_tar_to_buffer():
 
     return buffer
 
+def verify_installation(package, checksum_keys):
```

Comment:
Add type hints

Reply:
Is this necessary? Everything else doesn't have typehints and it's not something that users are expected to use.